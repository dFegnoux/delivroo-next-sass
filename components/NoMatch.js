import React from "react";
import "./noMatch.scss";

class NoMatch extends React.PureComponent {
  render() {
    return <h1 className="noMatch">404 !</h1>;
  }
}

export default NoMatch;
