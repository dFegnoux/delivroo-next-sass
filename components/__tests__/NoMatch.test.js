import React from "react";
import renderer from "react-test-renderer";
import NoMatch from "../NoMatch";

describe("NoMatch snapshots", () => {
  it("should render correctly with default props", () => {
    const tree = renderer.create(<NoMatch />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
