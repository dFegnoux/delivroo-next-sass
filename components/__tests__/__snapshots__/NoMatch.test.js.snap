// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch snapshots should render correctly with default props 1`] = `
<h1
  className="noMatch"
>
  404 !
</h1>
`;
